Embedded Metadata.js

```diff
@@ -680,11 +680,26 @@ function getAuthorFromByline(doc, newItem) {
 	}
 	
 	if(actualByline) {
+		//Z.debug(actualByline.innerHTML)
 		byline = ZU.trimInternal(actualByline.textContent);
-		Z.debug("Extracting author(s) from byline: " + byline);
-		byline = byline.split(/\bby[:\s]+/i);
-		byline = byline[byline.length-1].replace(/\s*[[(].+?[)\]]\s*/g, '');
-		var authors = byline.split(/\s*(?:(?:,\s*)?and|,|&)\s*/i);
+		var authors = [];
+		//make sure we're separating multiple authors on same line in byline
+		if (ZU.xpath(actualByline, './/*[string-length(text())>1]').length > 2){
```

Comment:
Well, not split per se, but we can see if the byline contains li elements and iterate through those.